README.md

```diff
@@ -18,6 +18,41 @@ slog.Info(ctx, "Loading widget", map[string]interface{}{
 })
 ```
 
+### Errors
+
+`slog` also provides a shorthand for capturing errors in the form of:
+
+```go
+slog.Error(ctx, "Failed to load widget", err)
+```
+
+This is functionally equivalent to:
+
+```go
+slog.Error(ctx, "Failed to load widget", map[string]interface{}{
```

Comment:
I know this PR has been merged already, and I'm not on my laptop so I can't try it out myself right now, but do the following work? I think they might, but they aren't mentioned in the readme and I don't think are tested explicitly, so it's hard for me to tell.

```go
slog.Error(ctx, "got an error: %v", map[string]interface{}{
    "error": err
})
```

```go
slog.Error(ctx, "got an error: %v", map[string]string{
    "error": err
})
```

I think these two – ie. the `error` param being part of the format string – will be what you find a great deal of in wearedev today, and what seems most natural to longtime users of slog. _Not_ supporting this convention will miss a lot of value, I think.